s2c_sensor_module/src/config/conf_can.h

```diff
@@ -70,7 +70,7 @@
  * quanta which means the bit rate is 8MHz / 16 = 500KHz.
  */
 /* Nominal bit Baud Rate Prescaler */
-#define CONF_CAN_NBTP_NBRP_VALUE    5
+#define CONF_CAN_NBTP_NBRP_VALUE    1//This value has been changes from 5 to 1

```

Comment:
No need for this comment. Value of 5 was a bug, no need to be reminded of that ;)